doc/cephfs/nfs.rst

```diff
@@ -79,3 +79,219 @@ Current limitations
 
 - Per running ganesha daemon, FSAL_CEPH can only export one Ceph file system
   although multiple directories in a Ceph file system may be exported.
+
+Exporting over NFS clusters deployed using rook
+===============================================
+
+In this tutorial we will be using minikube for setting up a single node test
+cluster. If you already have a kubernetes cluster deployed, please directly
+start with rook part.
+
+.. note:: Configuration of this tutorial should not be used in a a real
+          production cluster. For the purpose of simplification, the security
+          aspects of Ceph are overlooked in this setup.
+
+Minikube Setup
+--------------
+
+Installation::
+
+        dnf install kubernetes-client docker-common git qemu-kvm
```

Comment:
You may want to indicate which distro and relevant software package versions you're using.